network-api/networkapi/templates/fragments/footnotes.html

```diff
@@ -0,0 +1,22 @@
+{% load i18n wagtailcore_tags %}
+
+<div class="container mt-5 py-4">
+  <div class="row">
+    <div class="col-sm-10 offset-sm-1">
+      <hr>
+    </div>
+    <div class="col-sm-8 offset-sm-2 mt-5">
+      <h3 class="footnotes-title">{% trans "Footnotes" %}</h3>
+      {% for footnote in page.footnotes_list %}
+        <div class="footnote-row d-flex" id="footnote-{{ forloop.counter }}">
+          <a href="#footnote-source-{{ forloop.counter }}" class="footnote-number mr-3" aria-label="Back to content">
```

Comment:
Let’s translate accessibility labels, too

```suggestion
          <a href="#footnote-source-{{ forloop.counter }}" class="footnote-number mr-3" aria-label="{% trans"Back to content" %}">
```